admin-dev/themes/new-theme/scss/pages/product/product_page.scss

```diff
@@ -1149,6 +1149,43 @@ $product-page-padding-bottom: 80px !default;
     }
   }
 
+  #product_description_packed_products {
+    .search-with-icon {
+      max-width: 377px;
+    }
+
+    .entities-list-container {
+      display: flex;
+      flex-direction: row;
+      flex-wrap: wrap;
+      margin: 0.5rem -0.5rem;
```

Comment:
![image](https://user-images.githubusercontent.com/14963751/159229097-ecaab2d9-54e3-4348-9f29-72c9730cdd38.png)

Looks really close to the row class of bootstrap, maybe this is enough, by adapting a bit `.packed-product`

Also, by default a display flex is using `flex-direction: row`, it was probably useless to set it there